WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -380,7 +380,17 @@ private async Task ReadyToSignAsync(IEnumerable<AliceClient> aliceClients, DateT
 				{
 					await Task.Delay(delay, cancellationToken).ConfigureAwait(false);
 				}
-				await aliceClient.ReadyToSignAsync(cancellationToken).ConfigureAwait(false);
+
+				try
+				{
+					await aliceClient.ReadyToSignAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
@lontivero Do we keep retrying until the phase is still alive? Would be a bummer if we'd fail tx signing of all things because of a random network error.

Reply:
This PR is not for that. I have just created an issue for what you asked here: https://github.com/zkSNACKs/WalletWasabi/issues/8500